hwdb.d/60-sensor.hwdb

```diff
@@ -124,14 +124,12 @@ sensor:modalias:acpi:MXC6655*:dmi:*:svnAquarius*:pnNS483:*
 # Archos
 #########################################
 sensor:modalias:acpi:SMO8500*:dmi:*:svnARCHOS:pnARCHOS80Cesium:*
+sensor:modalias:acpi:KIOX000A*:dmi:*:svnARCHOS:pnARCHOS101CESIUM:*
```

Comment:
Could you add a brief comment here?
`# Archos 101 Cesium`
(Also for the first entry if possible, that I guess Archos 80 Cesium ??)

Reply:
Sure I can !